### DISCUSSION TITLE
ZNP Sonoff ZbBridgePro with zigbee2mqtt adapter errors

### DISCUSSION THREAD

**Discussion by el-bakkali** [arendst/Tasmota#20846] [Support request]
Hi all
im trying to use my ZbBridgePro with my docker install.

My config yalm includes :
serial:
  port: tcp://10.0.50.134:8888

My ZbBridgePro is running Tasmota 13.4.0.2 with Sonoff ZBPro TCP config

I see in the tasmota console its able to connect when i start the docker container 

20:37:23.724 TCP: Got connection from 10.0.10.140
20:38:33.030 TCP: Got connection from 10.0.10.140
20:39:42.237 TCP: Got connection from 10.0.10.140
20:40:51.592 TCP: Got connection from 10.0.10.140
20:42:00.802 TCP: Got connection from 10.0.10.140

However checking the container logs of zigbee2mqtt i get the following errors:

zigbee2mqtt  | Zigbee2MQTT:info  2024-03-01 19:43:10: Starting zigbee-herdsman (0.33.8)
zigbee2mqtt  | Zigbee2MQTT:error 2024-03-01 19:43:18: Error while starting zigbee-herdsman
zigbee2mqtt  | Zigbee2MQTT:error 2024-03-01 19:43:18: Failed to start zigbee
zigbee2mqtt  | Zigbee2MQTT:error 2024-03-01 19:43:18: Check https://www.zigbee2mqtt.io/guide/installation/20_zigbee2mqtt-fails-to-start.html for possible solutions
zigbee2mqtt  | Zigbee2MQTT:error 2024-03-01 19:43:18: Exiting...
zigbee2mqtt  | Zigbee2MQTT:error 2024-03-01 19:43:18: Error: SRSP - SAPI - readConfiguration after 6000ms
zigbee2mqtt  |     at Object.start (/app/node_modules/zigbee-herdsman/src/utils/waitress.ts:63:23)
zigbee2mqtt  |     at /app/node_modules/zigbee-herdsman/src/adapter/z-stack/znp/znp.ts:312:45
zigbee2mqtt  |     at Queue.execute (/app/node_modules/zigbee-herdsman/src/utils/queue.ts:35:26)
zigbee2mqtt  |     at Znp.request (/app/node_modules/zigbee-herdsman/src/adapter/z-stack/znp/znp.ts:300:27)
zigbee2mqtt  |     at ZnpAdapterManager.determineStrategy (/app/node_modules/zigbee-herdsman/src/adapter/z-stack/adapter/manager.ts:120:44)
zigbee2mqtt  |     at processTicksAndRejections (node:internal/process/task_queues:95:5)
zigbee2mqtt  |     at ZnpAdapterManager.start (/app/node_modules/zigbee-herdsman/src/adapter/z-stack/adapter/manager.ts:63:26)
zigbee2mqtt  |     at Controller.start (/app/node_modules/zigbee-herdsman/src/controller/controller.ts:132:29)
zigbee2mqtt  |     at Zigbee.start (/app/lib/zigbee.ts:60:27)
zigbee2mqtt  |     at Controller.start (/app/lib/controller.ts:109:27)

Any idea what is going on and why it wont work

I have ZB-GW03 and it works just fine
all i have to do is add to config yaml and it works, obviously i cant add that for the ZbBridgePro because its a diifferent adapter.
adapter: ezsp

what am i doing wrong ?

**Comment by sfromis** [comment]
The ZbBridgePro has a different Zigbee chip, a CC2652. That's usually called znp instead of ezsp from the ZB-GW03. I'm not sure, but I think that znp would be `adapter: zstack` in Zigbee2MQTT, otherwise similar to your working ezsp setup.

**Reply by el-bakkali** [reply]
unfortunately that doesnt work, i still get the same errors
<img width="399" alt="Screenshot 2024-03-01 at 19 59 14" src="https://github.com/arendst/Tasmota/assets/29339586/4ef030d9-a636-4c27-b88b-8574f3c48aac">

**Reply by el-bakkali** [reply]
ive tried removing the adapter as per other posts ive seen but it still dont work
